sdl2/pdcsetsc.c

```diff
@@ -92,14 +92,10 @@ int PDC_set_bold(bool boldon)
     if (!SP)
         return ERR;
 
-#ifdef PDC_WIDE
```

Comment:
Can you please try to rework the PR letting as much !PDC_WIDE code in as possible (this for example has zero effect, so definitely should be kept in)?
Doing so should keep the diff actually smaller and makes reviewing the patch easier.
I guess that in the end there won't be much questionable !PDC_WIDE parts - in this case we can work on those and drop one of the big "con" issues of this PR.